lib/core/utils/range_to_string.h

```diff
@@ -9,11 +9,13 @@ namespace clang {
 class ASTContext;
 class Expr;
 class Stmt;
+class Decl;
 }  // namespace clang
 
 namespace oklt {
 std::string getSourceText(const clang::SourceRange& range, clang::ASTContext& ctx);
 std::string getSourceText(const clang::Expr& expr, clang::ASTContext& ctx);
+std::string getSourceText(const clang::Decl& decl);
```

Comment:
We know that `Decl` includes `ASTContext`. But It would be nice to have API similar to all the others.

Reply:
It would be nice, but it's impossible. Some Stmts can have AST context. For example, you can get Decl from DeclRefExpr, but here is no unified way to do that, and sometimes there is no way to do that. :cry:

Reply:
Do you suggest adding a useless ASTContext parameter? The whole purpose of adding this was to make code simpler by removing the ast context for getSourceText for Decl.

Reply:
> Do you suggest adding a useless ASTContext parameter? The whole purpose of adding this was to make code simpler by removing the ast context for getSourceText for Decl.

Yes. Even if it's marked as `[[maybe_unused]]` it should still unify all the `getSourceText` functions under the same or similar API.

Reply:
Removed getSourceText(Decl)